### ISSUE TITLE
[1.12.2] Unable to craft Lapis Bocks, Light Blue Dye, Lime Dye in Crafting Grid if Erebus is installed.

### ISSUE THREAD

**Issue by Z-Tunic** [raoulvdberge/refinedstorage#2086]
Minecraft: 1.12.2
Forge: 14.23.5.2772

Mods: 2
Erebus-1.0.23
refinedstorage-1.6.9

-----

I should've made a post here a long time ago but forgot to do so, anyway... for some odd reason I am unable to craft certain recipes if Erebus is installed. Recipes for Lapis Blocks, Light Blue Dyes, Lime Dyes, etc. become 'blank' recipes. Not sure which other recipes are affected by this. Here's am image of what it looks like.
![](https://cdn.discordapp.com/attachments/377171551155650560/511851778753691650/refinedstorageerebusrecipebug.PNG)

I can craft them if I put them into a Vanilla Crafting Table. If I use a mod called "No more Recipe Conflict" and use the Crafting Gird... it'll show there being two possible recipes instead of just one recipe and if I cycle to the second recipe the item will show up, allowing me to craft it... only for it to reset back to the first 'blank' recipe. Here's an image of what it looks like.
![](https://cdn.discordapp.com/attachments/377171551155650560/511854037964947471/refinedstorageerebusrecipebugnomorerecipeconflict.PNG)

Just figure I give a post here to see if it's on Refined Storage's side or Erebus' side. On other Crafting Tables it still shows two recipes but the items are craft-able. If I remove Erebus then only one recipe shows and I'm able to craft the items.

**Comment by raoulvdberge** [comment]
Is erebus adding duplicate recipes?

**Comment by Z-Tunic** [comment]
No More Recipe Conflict shows there being two recipes instead of one, it could be possible. The Author of Erebus did mention about doing a fix in the recipe registry in the mod. Was hoping it was fixed in the next version of Erebus but... the issue still occurs.

I found the original issue about it. At the time it was posted I had foamfix added alongside Refined Storage & Erebus and thought Foamfix was the cause but it was narrowed down to just Refined Storage & Erebus.

https://github.com/vadis365/TheErebus/issues/603

I figure since the issue occurs still, it led me to give it a post here.
